L1Trigger/Phase2L1ParticleFlow/test/make_l1ctLayer1_dumpFiles_fromRAW_cfg.py

```diff
@@ -52,7 +51,7 @@
 process.PFInputsTask = cms.Task(
     process.TTClustersFromPhase2TrackerDigis,
     process.TTStubsFromPhase2TrackerDigis,
-    process.TrackerDTCProducer,
+    process.ProducerDTC,
```

Comment:
Given that every sub-detector in CMS has DTCs, is it reasonable to use the name ProducerDTC. which gives no clue which subdetector it's for?